app/deliver_grant_funding/admin/forms.py

```diff
@@ -110,6 +113,84 @@ def __init__(
         ]
 
 
+class PlatformAdminCreateGrantRecipientUserForm(FlaskForm):
+    users_data = TextAreaField(
+        "Grant recipient users TSV data",
```

Comment:
While playing around with this I found myself making small corrections and wanting to re-tab which shifts focus, we could do a little workaround to let you add the tab character to a textarea for inputs accepting TSV but I'm guessing the workflow we want to encourage is to get it right in the spreadsheet and then bring it over - trying to change it might be a dev only problem